be/src/connector/hive_connector.cpp

```diff
@@ -273,6 +273,85 @@ void HiveDataSource::_init_counter(RuntimeState* state) {
     }
 }
 
+static void build_nested_fields(const TypeDescriptor& type, const std::string& parent, std::string* sb) {
+    DCHECK(type.is_struct_type());
+    for (int i = 0; i < type.children.size(); i++) {
+        if (type.selected_fields[i]) {
```

Comment:
In this PR https://github.com/StarRocks/starrocks/pull/16469/files#, It remove the selected_fields

Reply:
OK, thanks. I'll merge this PR after that one, and see what's the final decision.